include/envoy/upstream/load_balancer_type.h

```diff
@@ -58,6 +58,11 @@ class LoadBalancerSubsetInfo {
    * fraction of hosts removed from the original host set.
    */
   virtual bool scaleLocalityWeight() const PURE;
+
+  /*
+   * @return bool whether to pick any host if the fallback policy fails to find a host.
```

Comment:
maybe something like `whether to attempt to select a host from the entire cluster if host selection from the fallback subset fails`

Reply:
Done.